### PR TITLE
add v1.0 release and fixed footer overlay on search results

### PR DESCRIPTION
**Pull request by thinksanky**
## Description ##
1. Changed the layout as per 
-https://sim.amazon.com/issues/search?q=is%3A(Open)+in%3A(60d338a9-21fb-4c44-8434-1d09e62adab0)&sort=score+desc&selectedDocument=35a2495e-6eeb-4419-82db-927fdb2b3ab1

2. Fixed the footer overlay that was covering the search results.

### CODE REVIEW

**Comment by eric-haibin-lin** [comment]
Was the footer overlay already fixed in https://github.com/apache/incubator-mxnet/pull/8877 ? @szha

File: docs/_static/mxnet-theme/index.html [hunk 1]
```diff
@@ -3,11 +3,7 @@
     <div class="row">
       <div class="col-lg-12">
           <div id="banner-title" class='col-sm-6 col-xs-12'><span>Apache MXNet</span>
-            <p id="landing-title">is a flexible and efficient library for deep learning.</p>
-            <p id="intro">
```
Comment (lupesko) [inline docs/_static/mxnet-theme/index.html]: Why did you remove this? we still want it...

File: docs/_static/mxnet-theme/index.html [hunk 2]
```diff
@@ -25,16 +21,16 @@
 <div class="section-tout">
   <div class="container">
     <div class="row">
+      <div class="col-lg-4 col-sm-12">
+          <h3>MXNet 1.0 Released</h3>
+          <p>We're excited to announce the release of MXNet 1.0! Check out the release notes for latest updates.</p>
+          <a href="https://github.com/dmlc/mxnet/releases">Learn More</a>
```
Comment (lupesko) [inline docs/_static/mxnet-theme/index.html]: This link should be: https://github.com/apache/incubator-mxnet/releases/tag/1.0.0

**Comment by szha** [comment]
footer layout problem was fixed in #8841

File: docs/_static/mxnet-theme/index.html (hunk 1 above)
Comment (thinksanky) [reply docs/_static/mxnet-theme/index.html]: This was per the previous PR I had done. - https://github.com/apache/incubator-mxnet-site/pull/33. 

Let me know when I have to add it back.

File: docs/_static/mxnet-theme/index.html (hunk 2 above)
Comment (thinksanky) [reply docs/_static/mxnet-theme/index.html]: Fixed this and pushed the PR.

File: docs/_static/mxnet-theme/index.html (hunk 1 above)
Comment (lupesko) [reply docs/_static/mxnet-theme/index.html]: 👍🏼

**Comment by piiswrong** [comment]
closing due to outdated
